### DISCUSSION TITLE
actions-sync push - x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by valenbb** [community/community#38230] [Enterprise]
Is there a way to bypass this error when not using a signed certificate for GitHub Enterprise Server editon?

C:\Temp\actions_sync\bin>actions-sync.exe push --cache-dir "/temp/azure_cache" --destination-token "xxx" --destination-url "https://IP-address"
not using impersonation for the requests
syncing `hashicorp/setup-terraform`
error creating github repository `hashicorp/setup-terraform`: error retrieving authenticated user: Get "https://IP-address/api/v3/user": x509: certificate signed by unknown authority

**Comment by ossanna16** [comment]
Hi there @valenbb and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
